extensions/amp-ooyala-player/0.1/validator-amp-ooyala-player.protoascii

```diff
@@ -0,0 +1,86 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-ooyala-player
+  tag_name: "SCRIPT"
+  spec_name: "amp-ooyala-player extension .js script"
+  mandatory_parent: "HEAD"
```

Comment:
Could you add 

`  also_requires_tag: AMP-OOYALA-PLAYER`

For new extensions, we want to disallow the extension javascript unless the extension tag is also present, since loading the javascript has a performance cost.